int level
string[] node_ids
string[] edges
std_msgs/Float32MultiArray entropy